config/samples/cdap_v1alpha1_cdapmaster.yaml

```diff
@@ -14,3 +14,7 @@ spec:
     metadata.storage.implementation: elastic
     metadata.elasticsearch.cluster.hosts: elasticsearch-master
     hdfs.user: root
+    security.enabled: "true"
+    security.auth.server.announce.address: ""
```

Comment:
Should use `security.auth.server.announce.urls` instead. The announce.address was deprecated a long while ago.